### DISCUSSION TITLE
I'd like to ask how I can find the documentation for version 5.3?

### DISCUSSION THREAD

**Discussion by hui200102** [fabricjs/fabric.js#10266] [Q&A]
I'd like to ask how I can find the documentation for version 5.3?

**Comment by drinkspiller** [comment]
https://web.archive.org/web/20240807094708/http://fabricjs.com/docs/index.html

**Comment by asturur** [comment]
You can also build and run the old website from the repo, or look at the old docs folder in the new website.
Some of those have been updated to run on 6.0.
I could deploy the old website maybe on something like fabricjs.com/5 i can try that, but i m not sure

**Reply by asturur** [reply]
https://fabricjs.com/fabricjs.com/demos/

Not sure how it works and i won't keep it forever for long, but if it helps you, here it is.

You will have to add that extra 'fabricjs.com' when it navigates out

**Comment by drinkspiller** [comment]
Some demos (e.g. https://fabricjs.com/fabricjs.com/kitchensink) would need updates to load and work properly
![image](https://github.com/user-attachments/assets/99d481f4-d6ec-4820-a100-03ae415d18ab)

**Reply by asturur** [reply]
Yes but time is what it is, is better to have a new kitchensink at some point or move the old website on a new org that can be deployed on its own small domain for reference.
